purchases/src/androidTestIntegrationTest/kotlin/com/revenuecat/purchases/Constants.kt

```diff
@@ -0,0 +1,13 @@
+package com.revenuecat.purchases
+
+object Constants {
+    const val API_KEY = "REVENUECAT_API_KEY"
+    const val PROXY_URL = "NO_PROXY_URL"
+    const val USER_ID = "integrationTest"
+    const val GOOGLE_PURCHASE_TOKEN = "GOOGLE_PURCHASE_TOKEN"
```

Comment:
I was able to make tests work by passing my own token I got in a test purchase. However, that token will probably fail in the backend after a few months (since Google's APIs won't return a response for it). Been thinking about how to do this other than refresh the token every few months. The only thing that comes to mind is to have an active non-test subscription running forever so we can keep it working. Thoughts?

Reply:
Yup, that's what I was thinking 👍